From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cjgronlund commented:
@LuisCabrer 
Thanks! Looks good.

carolinacmoravia committed:
M4847: Link that leads to the top of the page: [Packet capture analys… #13321

@gwallace,
[Packet capture analysis with Wireshark] link leads to the top of the page. Is this correct? Should it lead to another page?
Please, review this and sign off if you agree.
Many thanks in advance.

carolinacmoravia committed:
C52846: Broken bookmark: [Use DSC in disconnected environments]

@krnese, 
Could you review the reported bookmark? It is not working. 
Many thanks for your time in advance.

mimig1 committed:
Cosmos DB: Added info about columnar support.

mimig1 committed:
Added to the description.

mimig1 committed:
Small change.

Commit:
Merge pull request #17 from unifycloud/azure-database-security-best-practice

Add files via upload

v-nagta committed:
USER STORY 1044401 SaaS App Tutorial: Symantec Web Security Service (WSS)

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
updated according to client requirements

v-nagta committed:
updated

v-nagta committed:
USER STORY 1044436 SaaS App Tutorial: Zivver

shruja committed:
USER STORY 1045254: SaaS App Tutorial: Realtimeboard

shruja committed:
update

v-nagta committed:
updated

v-nagta committed:
updated

Commit:
Create azure-database-security-checklist.md

Commit:
Merge pull request #18 from unifycloud/azure-database-security-checklist

Create azure-database-security-checklist.md

Commit:
Update azure-database-security-checklist.md

Commit:
Update azure-database-security-checklist.md

Commit:
Update azure-database-security-checklist.md

Commit:
Update azure-database-security-checklist.md

Commit:
Create azure-operational-security-checklist.md

Commit:
Merge pull request #19 from unifycloud/azure-operational-security-checklist

Create azure-operational-security-checklist.md

Commit:
Update azure-operational-security-checklist.md

Commit:
Create azure-operational-security-overview.md

Commit:
Update azure-operational-security-checklist.md

Commit:
Update azure-operational-security-overview.md

Commit:
Update azure-operational-security-overview.md

Commit:
updates to app gateway articles

Commit:
Update azure-operational-security-overview.md

Commit:
Update azure-database-security-checklist.md

Commit:
Create iot-hub-raspberry-pi-kit-python-get-started.md

Commit:
Updating video URL in Using Node.js Modules article

Commit:
Updating metadata

Commit:
Fixing Acrolinx issues

rayne-wiselman committed:
new walkthrough articles

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-architecture.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-enable-replication.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-network.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-source-target.md

rayne-wiselman committed:
update toc

rayne-wiselman committed:
Merge branch 'rayne-july31' of https://github.com/rayne-wiselman/azure-docs-pr into rayne-july31

Commit:
Created tabs.

Commit:
Fixes.

Commit:
Fixes.

Commit:
Fix.

Commit:
updates

Commit:
updates to links

Commit:
corrected date

Commit:
Update iot-hub-raspberry-pi-kit-python-get-started.md

Commit:
Add files via upload

Commit:
Update iot-hub-raspberry-pi-kit-python-get-started.md

Commit:
Update TOC.md

Commit:
Update iot-hub-get-started.md

Commit:
Update iot-hub-raspberry-pi-kit-python-get-started.md

Commit:
Update iot-hub-get-started-physical.md

Commit:
Update iot-hub-raspberry-pi-kit-python-get-started.md

Commit:
Update iot-hub-raspberry-pi-kit-python-get-started.md

Commit:
Update iot-hub-raspberry-pi-kit-python-get-started.md

mimig1 committed:
Small updates to get this published.

Juliako committed:
Update media-services-error-codes.md

Commit:
updates to dns articles

Juliako committed:
Update media-services-error-codes.md

mimig1 committed:
Fixed on screen formatting of conn string.

mimig1 committed:
Another code formatting fix.

CarlRabeler committed:
Update sql-database-service-tiers-table-elastic-pools.md

JasonWHowell committed:
Update connect-python.md

GitHubber17 committed:
Merge pull request #18381 from georgewallace/dns0731

DNS : 07/31 custom domain updates

markgalioto committed:
creating new article - restore Windows Server system state

Commit:
Fixing still more Acrolinx issues

cherylmc committed:
add info about traffic selectors

Commit:
Fixing style issue

Commit:
Fixing even MORE Acrolinx issues (for an article I didn't write)

mimig1 committed:
Incorporated Govind's feedback

Commit:
update 1000 TB to 1 TB

Lisaco88 committed:
Merge pull request #18211 from unifycloud/azure-operational-security-overview

Azure operational security overview

Commit:
Merge pull request #18400 from cherylmc/patch-1

add info about traffic selectors

markgalioto committed:
fixing a broken link

cherylmc committed:
update warning

Commit:
Merge pull request #18403 from stevestein/patch-2

update 1000 TB to 1 TB

markgalioto committed:
updating Backup TOC to include the restore system state article

GitHubber17 committed:
Merge pull request #18341 from rmcmurray/20170730

Updating video URL in Using Node.js Modules article

cherylmc committed:
update warning

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into routingupdate

CarlRabeler committed:
Update sql-database-elastic-pool.md

spboyer committed:
404 Error on link

Commit:
Merge pull request #18408 from spboyer/patch-5

404 Error on link

markgalioto committed:
adding the restore article to the Azure Backup TOC

GitHubber17 committed:
Merge pull request #18364 from shizn/patch-21

add python pi tutorials

Commit:
Merge pull request #18397 from JasonWHowell/patch-1

PostgreSQL: Update connect-python.md

curtand committed:
add classic note

GitHubber17 committed:
Merge pull request #18208 from unifycloud/azure-database-security-checklist

Azure database security checklist

markgalioto committed:
fixing broken link

Commit:
fixes

Commit:
Made some minor changes for rli

GitHubber17 committed:
Merge pull request #18405 from cherylmc/routingupdate

update warning

Commit:
removed images

Commit:
Merge pull request #18410 from CamSoper/master

Made some minor changes for rli

Commit:
Merge pull request #18378 from Juliako/patch-8

Update media-services-error-codes.md

markgalioto committed:
updating some metadata in articles

Juliako committed:
Update media-services-frequently-asked-questions.md

GitHubber17 committed:
Merge pull request #18261 from georgewallace/appgatewayupdates

app gateway : updates to app gateway articles

Lisaco88 committed:
Merge pull request #18210 from unifycloud/azure-operational-security-checklist

Azure operational security checklist

GitHubber17 committed:
Merge pull request #18112 from shruja/realtimeboard

USER STORY 1045254: SaaS App Tutorial: Realtimeboard

GitHubber17 committed:
Merge pull request #18110 from v-nagta/zivver

USER STORY 1044436 SaaS App Tutorial: Zivver

GitHubber17 committed:
Merge pull request #18012 from v-nagta/symantecupdate

USER STORY 1044401 SaaS App Tutorial: Symantec Web Security Service (…

Juliako committed:
Update media-services-frequently-asked-questions.md

Juliako committed:
Update media-services-rest-configure-asset-delivery-policy.md

Juliako committed:
Update media-services-frequently-asked-questions.md

Juliako committed:
Update media-services-rest-configure-content-key-auth-policy.md

Juliako committed:
Update media-services-rest-create-contentkey.md

Juliako committed:
Update media-services-rest-dynamic-manifest.md

Juliako committed:
Update media-services-rest-encode-asset.md

Juliako committed:
Update media-services-rest-get-media-processor.md

Juliako committed:
Update media-services-rest-manage-entities.md

Juliako committed:
Update media-services-rest-storage-encryption.md

Juliako committed:
Update media-services-rest-upload-files.md

Commit:
Merge pull request #18413 from Juliako/patch-8

Update media-services-frequently-asked-questions.md

Commit:
Merge pull request #18420 from Juliako/patch-14

Update media-services-rest-create-contentkey.md

Commit:
Merge pull request #18421 from Juliako/patch-15

Update media-services-rest-dynamic-manifest.md

Commit:
Merge pull request #18423 from Juliako/patch-17

Update media-services-rest-get-media-processor.md

Commit:
Merge pull request #18424 from Juliako/patch-18

Update media-services-rest-manage-entities.md

Juliako committed:
Update media-services-dotnet-get-started.md

Commit:
Merge pull request #18425 from Juliako/patch-19

Update media-services-rest-storage-encryption.md

Commit:
Merge pull request #18426 from Juliako/patch-20

Update media-services-rest-upload-files.md

Juliako committed:
Update media-services-get-media-processor.md

Juliako committed:
Update media-services-dotnet-encode-with-media-encoder-standard.md

Juliako committed:
Update media-services-motion-detection.md

Commit:
Merge pull request #18418 from Juliako/patch-12

Update media-services-rest-configure-asset-delivery-policy.md

Commit:
Merge pull request #18419 from Juliako/patch-13

Update media-services-rest-configure-content-key-auth-policy.md

Commit:
Merge pull request #18422 from Juliako/patch-16

Update media-services-rest-encode-asset.md

Commit:
Merge pull request #18427 from Juliako/patch-8

Update media-services-dotnet-get-started.md

Juliako committed:
Update media-services-process-content-with-indexer2.md

Commit:
Merge pull request #18428 from Juliako/patch-14

Update media-services-get-media-processor.md

Juliako committed:
Update media-services-video-optical-character-recognition.md

Commit:
Merge pull request #18429 from Juliako/patch-15

Update media-services-dotnet-encode-with-media-encoder-standard.md

Commit:
Merge pull request #18430 from Juliako/patch-17

Update media-services-motion-detection.md

Juliako committed:
Update media-services-face-redaction.md

Juliako committed:
Update media-services-face-redaction.md

Juliako committed:
Update indexer-task-preset.md

Commit:
Merge pull request #18432 from Juliako/patch-8

Update media-services-video-optical-character-recognition.md

Commit:
Merge pull request #18431 from Juliako/patch-18

Update media-services-process-content-with-indexer2.md

Commit:
Merge pull request #18433 from Juliako/patch-10

Update media-services-face-redaction.md

Commit:
Merge pull request #18434 from Juliako/patch-11

Update indexer-task-preset.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-architecture.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-architecture.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-network.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-network.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-replication.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-test-failover.md

rayne-wiselman committed:
Update TOC.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-prerequisites.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-overview.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-network.md

Commit:
Changed "create subnet" to "add subnet"

Commit:
Changes

Commit:
Format fix

Commit:
Format fix.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vnet-classic-c

Commit:
Removed tabs.

Commit:
Fixed IP version links.

Commit:
Changes.

Commit:
Update.

Commit:
Merge pull request #18440 from jimdial/patch-98

Fixed IP version links.

Commit:
Link fix.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vnet-classic-c

AnoopVasudavan committed:
addign linux push install

georgewallace committed:
corrected link

carolinacmoravia committed:
Merge pull request #2 from georgewallace/patch-5

corrected link

carolinacmoravia committed:
Update network-watcher-alert-triggered-packet-capture.md

Commit:
Merge pull request #18442 from AnoopVasudavan/master

addign linux push install

Commit:
Merge pull request #14997 from carolinacmoravia/patch-246

M4847: Link that leads to the top of the page: [Packet capture analys…

Commit:
Merge pull request #18383 from CarlRabeler/patch-313

Update sql-database-service-tiers-table-elastic-pools.md

carolinacmoravia committed:
Update site-recovery-automate-mobility-service-install.md

Commit:
Merge pull request #16485 from carolinacmoravia/patch-254

C52846: Broken bookmark: [Use DSC in disconnected environments]

markgalioto committed:
minor edits and fixing remaining blocking bug

Lisaco88 committed:
Merge pull request #18350 from jimdial/vnet-classic-c

Created tabs.

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-network.md

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-architecture.md

markgalioto committed:
one last tweak of the formatting

Lisaco88 committed:
Merge pull request #18438 from jimdial/patch-97

Changed "create subnet" to "add subnet"

rayne-wiselman committed:
Update vmm-to-vmm-walkthrough-network.md

matt1883 committed:
[ADLA] PowerShell manage - Pipeline, recurrence, compute policy content

Added a section for pipeline and recurrence info
Added a section for compute policy info

Lisaco88 committed:
Merge pull request #18342 from rayne-wiselman/rayne-july31

new walkthrough articles

mimig1 committed:
editorial change

Commit:
Merge pull request #18446 from matt1883/patch-1

[ADLA] PowerShell manage - Pipeline, recurrence, compute policy content

Lisaco88 committed:
Merge pull request #18399 from markgalioto/Az7-31b

creating new article - restore Windows Server system state

Commit:
Merge pull request #16793 from mimig1/07062017l2

Cosmos DB: Added info about columnar support.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

markgalioto committed:
fixing link issues and minor updates for upgrade backup vault

v-alje committed:
Merge pull request #18449 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #18409 from curtand/class0731

add classic note

Commit:
Merge pull request #18450 from markgalioto/Az8-1

fixing link issues and minor updates for upgrade backup vault
